tests/cpp/operator/mkldnn.cc

```diff
@@ -1306,6 +1326,97 @@ void TestOpEx(const OpAttrs &forward_attrs, const OpAttrs &backwards_attrs) {
   }
 }
 
+// compares output of fcompute with fcomputex
+void TestSoftmaxOp(const OpAttrs &forward_attrs) {
```

Comment:
cant' we just reuse TestOpEx but make the backwards_op optional?

Reply:
Yes, I am planning to do reuse TestOpEx and remove TestSoftmaxOp. I implemented it as a separate function just for POC to verify my logic. Forgot to mention it in the description.